### PR TITLE
Query Job status

### PR DESCRIPTION
**Pull request by Ridhwana**
In this PR, we add a Utilities Class, this will be the basis for all Smile Identity Utility functions. The first method (get_job_status) is now being used by web api to get the job status. The polling however still remains as part of the web api class.
Also updated the documentation accordingly.

giphy (8)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,9 +143,46 @@ The response will be a stringified json object:
...
+```
+
+#### Utilities Class
+
+You may want to receive more information about a job. This is built into Web Api if you choose to set return_job_status as true in the options class. However, you also have the option to build the functionality yourself by using the Utilities class. Please note that if you are querying a job immediately after submitting it, you will need to poll it for the duration of the job.
+
+```java
+import smile.identity.core.Utilities;
+
+String job_status = new Utilities(<partner_id>, <the decoded-version of-your-api-key>, <sid_server>).get_job_status(<user_id>, <job_id>, <return_image_links> , <return_history>);
```
Comment (beaglebets) [inline README.md]: It would probably behoove us to make the job_status accept an options hash (or object or whatever it needs to be in Java)

Comment (beaglebets) [inline README.md]: Bill had a rather strenuous objection to the Utilities class idea on Friday. Before approving this structure we probably should discuss with him, as he was adamant that we were completely wrong on this concept for some reason.

Comment (Ridhwana) [reply README.md]: I've kept the Utilities class but used it to expose a method on the Web api class

Comment (Ridhwana) [reply README.md]: Amended

**Pull request merged**